package.json

```diff
@@ -20,7 +31,7 @@
     "publish:p2p": "f () { lerna publish from-package ;}; f",
     "serve": "f () { lerna exec --scope=@deriv/${1:-'*'} -- npm run serve $npm_config_open;}; f",
     "start": "f () { lerna exec --scope=@deriv/${1:-'*'} -- npm run start ;}; f",
-    "test": "f () { bash ./scripts/circleci-config.test.sh && lerna exec --scope=@deriv/${1:-'*'} --parallel -- npm run test ;}; f",
+    "test": "f () { bash ./scripts/circleci-config.test.sh && eslint \"./packages/*/src/**/*.?(js|jsx)\" ;}; f",
```

Comment:
I think we still want to run `npm run test` each package? since some packages also run mocha tests etc.

Reply:
You are so right, I totally missed it.